sdk/android/skin/src/main/java/com/ooyala/android/skin/OoyalaSkinLayoutController.java

```diff
@@ -296,6 +297,17 @@ private void injectLocalizedResources(JSONObject configJson, Context context) {
     return languageFiles;
   }
 
+  private void setDefaultAudioLanguage(JSONObject audioLanguage) {
```

Comment:
Please document all your methods. Even if they are private.  It's in our code style guide and will avoid making more code debt.

Reply:
Sure. Thanks. I'll do it on Monday